components/worktemplates/components/preview/section.scss

```diff
@@ -0,0 +1,84 @@
+@import 'utils/variables';
```

Comment:
The rest of the feature uses styled-components, it does not really make sense to start using external CSS file in the middle of it 🤔